plugins/provenance/server/resource.py

```diff
@@ -394,7 +394,13 @@ def fileRemoveHandler(self, event):
         if 'itemId' not in file:
```

Comment:
Why?  Files can either have `itemId`s or be `attachedToId`.  If they have an `itemId`, we add provenance to the item.  If they don't, we don't.  When a file with `attachedToId` is removed, we shouldn't even be trying to load an item (even if that would fail, which it might do otherwise, for instance, if the item were deleted before the file).